New check of description is causing other ros package to fail

We are seeing errors running with the latest version of dynamic_reconfigure as the new check is causing failures.   The latest version of ros image_transport_plugins (1.9.5) fails with the following error:

 ```
File "/tmp/buildd/workspace/dynamic_reconfigure/parameter_generator_catkin.py", line 70, in check_description
raise Exception(r"""quotes not allowed in description string `%s`""" % description)
Exception: quotes not allowed in description string `Try to achieve either 'target_bitrate' or 'quality'`
make[4]: *** [/tmp/buildd/workspace/theora_image_transport/include/theora_image_transport/TheoraPublisherConfig.h] Error 1
```

Is there a way to allow for backward compatibility or before turning the check on ensure that all other packages are compliant?

Any help with this would be appreciated.

@mikepurvis

--- a/src/dynamic_reconfigure/parameter_generator_catkin.py
+++ b/src/dynamic_reconfigure/parameter_generator_catkin.py
@@ -141,7 +141,6 @@ class ParameterGenerator:
             if type == str_t and (max != None or min != None):
                 raise Exception("Max or min specified for %s, which is of string type"%name)
             check_name(name)
-            check_description(description)
             self.gen.fill_type(newparam)
             self.gen.check_type_fill_default(newparam, 'default', self.gen.defval[paramtype])
             self.gen.check_type_fill_default(newparam, 'max', self.gen.maxval[paramtype])

